### PR TITLE
version update and template update

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@codiebeulaine please review cc @miltontony

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (codiebeulaine) [inline .cache/v/cache/lastfailed]: this file is not needed, please remove

**Comment by g1039** [comment]
Ready for re-review @codiebeulaine

**Comment by g1039** [comment]
@codiebeulaine Ready for review cc @miltontony

**Comment by codiebeulaine** [comment]
:+1:

**Pull request merged**